Documentation/ceph-common-issues.md

```diff
@@ -817,3 +818,70 @@ If you still decide to configure an OSD on LVM, please keep the following in min
 - Avoid incrementing the `count` field of `storageClassDeviceSets` and create a new LV that backs a OSD simultaneously.
 
 You can know whether the above-mentioned tag exists tag with the command: `sudo lvs -o lv_name,lv_tags`. If the `lv_tag` field is empty in an LV corresponding to the OSD lv_tags, this OSD encountered the problem. In this case, please [retire this OSD](ceph-osd-mgmt.md#remove-an-osd) or replace with other new OSD before restarting.
+
+## The Ceph-CSI driver was being unexpectedly removed
+
+### Symptoms
+
+Sometimes the Ceph-CSI driver is unexpectedly removed by the garbage collector. For more detail, please refer to [this issue](https://github.com/rook/rook/issues/6162).
```

Comment:
This need rewords as the GC will try to  get the ownerRef of the CSIDriver but it fails to GET resource as the ownerRef is not cluster scoped resource and triggers garbage collection of resources owned by deployment `rook-ceph-operator`